.github/workflows/cleanup-test-user-workspaces.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         if: always()
         uses: actions/upload-artifact@v2
         with:
-          name: logs-and-context-[${{ matrix.testOptions }}]
+          name: logs-and-context
```

Comment:
what is this change for?

Reply:
The nightly tests GH action uses a matrix to run tests against different servers. Then it zips up the logs from each test run and uploads them as GH artifacts with the name `logs-and-context-[server_name]`. This GH action doesn't use a matrix and always runs against the `terra-dev` server, so I just removed the suffix.